Added some more details on creating color schemes with examples (#103)
MicrosoftDocs/terminal: TerminalDocs/customize-settings/color-schemes.md

description: Learn how to create color schemes for Windows Terminal.
author: cinnamon-msft
ms.author: cinnamon
ms.date: 7/28/2020
ms.topic: how-to
ms.service: terminal
ms.localizationpriority: high
---

# Color schemes in Windows Terminal

> Windows Terminal lets you define your own color schemes, either by using the built-in preset themes, or by creating your own theme from scratch. To change  themes, you'll need to edit the settings.json file in an editor such as [Visual Studio Code](https://code.visualstudio.com/).

Reviewer: ```suggestion
Windows Terminal lets you define your own color schemes, either by using the built-in preset schemes, or by creating your own scheme from scratch. To change schemes, you'll need to edit the settings.json file in an editor such as [Visual Studio Code](https://code.visualstudio.com/).
```

Reviewer: For context, we'll be adding themes later, so this will help avoid confusion in the future. 😊

## Switching to a different color scheme

> Launch the Windows Terminal, and then select the small downward-facing arrow in the title bar. This will open a pull-down menu that lists the available profiles on your system (for example, Windows PowerShell and Command Prompt) and some other options. Select **Settings**, and the settings.json file will open in your default text editor.

Reviewer: ```suggestion
Launch Windows Terminal and then select the small downward-facing arrow in the title bar. This will open a pull-down menu that lists the available profiles on your system (for example, Windows PowerShell and Command Prompt) and some other options. Select **Settings**, and the settings.json file will open in your default text editor.
```

This file is where you can define various options per window or per profile. To demonstrate, let's change the color scheme for the Command Prompt profile.

Look down the JSON file until you find the section that includes:

```json
"commandline": "cmd.exe",
"hidden": false"
```

Change it to read:

```json
"commandline": "cmd.exe",
"hidden": false",
"colorScheme": "Tango Light"
```

Notice the extra comma in the **hidden** line. Once you save this file, Windows Terminal will update any open window. Open a Command Prompt tab if you haven't already, and you'll immediately see that the colors have changed.

> 

Reviewer: ```suggestion
```

### Tango Light

![Windows Terminal Tango Light color scheme](./../images/tango-light-color-scheme.png)

## More schemes

> For more schemes, see the [Custom Terminal Gallery](../custom-terminal-gallery/custom-schemes.md) section.

Reviewer: This should change depending on what the page is renamed to.
